src/core_modules/capture-ui/UserAvatarImage/UserAvatarImage.component.js

```diff
@@ -0,0 +1,56 @@
+// @flow
```

Comment:
Can you also change the name of this file and its parent folder to `CardImage`? Thanks!